src/components/menu.js

```diff
@@ -66,16 +66,36 @@ class Menu extends Component {
         <div>
           <ul>
             <li>
-              <Link onClick={()=>{this.onClickClose()}} to="/">blog</Link>
+              <Link
+                onClick={() => {
+                  this.onClickClose();
```

Comment:
Creo que esto puede quedar mejor así: 
```suggestion
onClick={this.onClickClose}
```